README.md

```diff
@@ -119,6 +119,10 @@ aws:
     attribute_names:
       - ApproximateReceiveCount
       - SentTimestamp
+sqs:
+  log_level: debug
+sns:
+  log_level: info
```

Comment:
The [`log_level`](http://docs.aws.amazon.com/AWSRubySDK/latest/AWS/Core/Configuration.html#log_level-instance_method) configuration option configures the log level that all AWS SDK API call messages log at.

The default it logs at is `info`. This pollutes our `info.log` file (which logs `info` and above) with every single SQS request that is made, which is very nosy and makes the logs very cluttered. By configuring it to `debug` log level, the SQS SDK logging is no longer output to `info.log`, and instead goes to our specific `debug.log`.